pkg/api/meta/meta.go

```diff
@@ -321,16 +321,18 @@ func (resourceAccessor) SetResourceVersion(obj runtime.Object, version string) e
 // genericAccessor contains pointers to strings that can modify an arbitrary
 // struct and implements the Accessor interface.
 type genericAccessor struct {
-	namespace       *string
-	name            *string
-	generateName    *string
-	uid             *types.UID
-	apiVersion      *string
-	kind            *string
-	resourceVersion *string
-	selfLink        *string
-	labels          *map[string]string
-	annotations     *map[string]string
+	namespace         *string
```

Comment:
Code paths that use this are slow, I seem to recall @smarterclayton going to some effort to prevent usage of this?